openslides/core/projector.py

```diff
@@ -139,3 +139,10 @@ class Message(ProjectorElement):
     def check_data(self):
         if self.config_entry.get('message') is None:
             raise ProjectorException('No message given.')
+
+
+class ListOfSpeakersOverlay(ProjectorElement):
+    """
+    List of speakers (last, current, next) on the projector. Rendered as overlay.
+    """
+    name = 'core/speakeroverlay'
```

Comment:
Hm. If you want to display potentially all usernames on your projector element then you just have to leave them all out. I do not see any other possibility at the moment. Maybe we can later implement a lazy loading of the projector if some other stuff than the projector itself changes.